prov/rxd/src/rxd_ep.c

```diff
@@ -1155,6 +1155,23 @@ static void rxd_init_peer(struct rxd_ep *ep, uint64_t rxd_addr)
 	dlist_init(&ep->peers[rxd_addr].buf_pkts);
 }
 
+void rxd_ep_peers_grow(struct rxd_ep *ep, fi_addr_t rxd_addr) {
+
+	size_t i;
+	size_t new_max_peers;
+
+	if (rxd_addr == ep->max_peers) {
+		new_max_peers = 2 * ep->max_peers;
+		assert(new_max_peers > ep->max_peers);
+		ep->peers = realloc(ep, sizeof(struct rxd_peer) *
+				    new_max_peers);
+		assert(ep->peers);
+		for(i = ep->max_peers; i < new_max_peers; 
+		    rxd_init_peer(ep, i++));
```

Comment:
Please move the rxd_init_peer inside the for statement, rather than keeping it part of the for iteration.  This is for clarity as to what the for loop is doing, and avoids having the 'hidden' ; at the end of the for loop.